### PR TITLE
Removing JHU mentions 

### PR DESCRIPTION
**Pull request by cansavvy**
## Summary 

This PR is about removing only JHU relevant information from the website since DaSEH is not a JHU branded thing. 

I did this by: 

- Search and replacing `jhudatascience.org/intro_to_r with daseh.org`
- Searching for terms like "hopkins" and "JHU" and removing that content where it was not applicable. 
- On applicable information that probably needs replacements, I put `TODO:`s which I will now also make into GitHub issues. 

## Follow up question 

Will the `jhur` package need to be made into a DASEH flavor for use with this? That would be extra maintenance so it should only be done if it would add value. Alternatively its name could be changed so its clear its used here and at `intro_to_r`. OR nothing needs to be changed and that's fine too.

### CODE REVIEW

**Review by avahoffman — APPROVED** [review]
Fantastic! I think just correcting `daseh.org//something` to `daseh.org/something` and we are all set :) 

Thanks for also catching the JHU courses and evals!

re: `jhur` -- this is largely a package for convenience of loading data. Because DaSEH will have different datasets, it makes sense to maintain a different package.

**Comment by cansavvy** [comment]
Oh shoot. Must have been sloppy with /. That's the problem when I do this during a meeting. Will fix tonight.

**Comment by cansavvy** [comment]
I like daseh_data! Has a ring to it!

File: help.Rmd [hunk 1]
```diff
@@ -73,12 +73,12 @@ Hit the "esc" key to restore your `>`, then fix/rerun your code.
 ***
 ## **When should I use quotes ("") or backticks (``)?**
 
-Check out our [Guide on when to use quotes or backticks](https://jhudatascience.org/intro_to_r/resources/quotes_vs_backticks.html).
+Check out our [Guide on when to use quotes or backticks](https://daseh.org//resources/quotes_vs_backticks.html).
```
[the remark this answers is not in the document]
Comment (cansavvy) [reply help.Rmd:76]: Shoot. Whoops

[the remark this answers is not in the document]
Comment (cansavvy) [reply help.Rmd]: Fixed! Just searched and replaced `https://daseh.org//` with `https://daseh.org/`

**Review by avahoffman — APPROVED** [review]
Lgtm!

**Pull request merged**